en/mkdocs.yml

```diff
@@ -342,33 +340,35 @@ nav:
         - 'Rule Based Provisioning': using-wso2-identity-server/rule-based-provisioning.md
         - 'Provisioning Patterns': using-wso2-identity-server/provisioning-patterns.md
       - 'Account Management Policies':
+        - 'Account Management Policies': using-wso2-identity-server/account-Management-Policies.md
         - 'Account Recovery':
+          - 'Account Recovery': using-wso2-identity-server/account-Recovery.md
           - 'Username Recovery':
-            - 'Username Recovery': using-wso2-identity-server/username-recovery.md
-            - 'Configuring reCaptcha for Username Recovery': using-wso2-identity-server/configuring-recaptcha-for-username-recovery.md
+            - 'Username Recovery': using-wso2-identity-server/username-Recovery.md
+            - 'Configuring reCaptcha for Username Recovery': using-wso2-identity-server/configuring-reCaptcha-for-Username-Recovery.md
           - 'Password Recovery':
-            - 'Password Recovery': using-wso2-identity-server/password-recovery.md
-            - 'Configuring Google reCaptcha for Security-Question Based Password Recovery': using-wso2-identity-server/configuring-google-recaptcha-for-security-question-based-password-recovery.md
-            - 'Configuring reCaptcha for Password Recovery': using-wso2-identity-server/configuring-recaptcha-for-password-recovery.md
-          - 'Managing Challenge Questions': using-wso2-identity-server/managing-challenge-questions.md
+            - 'Password Recovery': using-wso2-identity-server/password-Recovery.md
+            - 'Configuring Google reCaptcha for Security-Question Based Password Recovery': using-wso2-identity-server/configuring-Google-reCaptcha-for-Security-Question-Based-Password-Recovery.md
+            - 'Configuring reCaptcha for Password Recovery': using-wso2-identity-server/configuring-reCaptcha-for-Password-Recovery.md
+          - 'Managing Challenge Questions': using-wso2-identity-server/managing-Challenge-Questions.md
```

Comment:
the .md file names need to be reverted back to simple case. please fix in all the relevant places